### PR TITLE
NativeAOT-LLVM: Enable CI for Windows/Debug

### PR DESCRIPTION
**Pull request by yowl**
This PR enables CI for the LLVM backend, initially by running just the Generics test for Windows build of Wasm in Debug configuration.

### CODE REVIEW

**Comment by MichalStrehovsky** [comment]
@yowl can you access the CI logs now? I asked around last week and the 404s you were hitting were apparently an Azure DevOps bug.

(I guess looking at the commits, I assume you can.)

**Comment by yowl** [comment]
@MichalStrehovsky Thanks, I can see all that I need to.

File: dotnet.cmd [hunk 1]
```diff
@@ -19,9 +19,4 @@ set DOTNET_MULTILEVEL_LOOKUP=0
 :: Disable first run since we want to control all package sources
 set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 
-:: Temporarily make this dotnet more permanent
```
Comment (jkotas) [inline dotnet.cmd:22]: I am going to submit merge from feature/NativeAOT to get these out of the way.

File: eng/pipelines/runtimelab/install-emscripten.cmd [hunk 2]
```diff
@@ -0,0 +1,24 @@
+mkdir "%1" 2>nul
+cd "%1"
+
+git clone https://github.com/emscripten-core/emsdk.git
```
Comment (jkotas) [inline eng/pipelines/runtimelab/install-emscripten.cmd:4]: @safern Could you please take a look at this and confirm that it is ok to do these sort of things on CI machines?

File: eng/pipelines/runtimelab/install-emscripten.cmd [hunk 3]
```diff
@@ -0,0 +1,24 @@
+mkdir "%1" 2>nul
+cd "%1"
```
Comment (jkotas) [inline eng/pipelines/runtimelab/install-emscripten.cmd:2]:
```suggestion
cd /D "%1"
```

**Review by jkotas — APPROVED** [review]
Thank you!

**Pull request merged**